packages/oruga/src/components/datepicker/Datepicker.vue

```diff
@@ -852,13 +852,15 @@ function formatNative(value: Date | Date[]): string {
 /** Parse string into date */
 function onChange(value: string): void {
     const date = (props.dateParser as any)(value, defaultDateParser);
+    const validDate = (d: unknown): d is Date =>
```

Comment:
What do you think about moving this to `utils/helper.ts` and name it `isDate` like the existing helper `isObject` ?